core/src/types/response.rs

```diff
@@ -115,7 +115,7 @@ impl From<Error> for Response {
 	fn from(error: Error) -> Self {
```

Comment:
This implementation is no longer valid. Should be more explicity: `Response::from_error(error, jsonrpc)`